packages/scaffold/src/views/w3m-connect-view/index.ts

```diff
@@ -200,15 +200,19 @@ export class W3mConnectView extends LitElement {
       return null
     }
 
-    const roundedCount = Math.floor(ApiController.state.count / 10) * 10
+    const count = ApiController.state.count
+    const featuredCount = ApiController.state.featured?.length
+    const rawCount = count + featuredCount
```

Comment:
could this result in a problem if `featuredCount` is undefined?.
If `ApiController.state.featured` is undefined, this breaks I think 🤔

Reply:
Not sure how I went with optional property access operator since it's not expected to be undefined